app/services/Config.scala

```diff
@@ -65,7 +68,6 @@ sealed trait Config {
   }
 
   lazy val permissionsStage: String = readTag("Stage") match {
-    case Some("DEV") =>  "CODE"
```

Comment:
So all this does is map the DEV stage of TagManager (Local) to the CODE stage of permissions. It's prefereable to do this, or we'd have to all run permissions locally to change local permissions